src/styles/main.scss

```diff
@@ -1,3 +1,45 @@
 body {
   margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 300px);
+  justify-content: center;
+}
+
+.container__block {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 300px;
+  min-width: 300px;
+  font-family: "Arial, Helvetica, sans-serif";
+  font-size: 100px;
+  color: white;
+
+  &:nth-child(1) {
+    background-color: rgb(255, 0, 0);
+  }
+  
+  &:nth-child(2) {
+    background-color: rgb(205, 0, 0);
+  }
+  
+  &:nth-child(3) {
+    background-color: rgb(154, 0, 0);
+  }
+  
+  &:nth-child(4) {
+    background-color: rgb(102, 0, 0);
+  }
+  
+  &:nth-child(5) {
+    background-color: rgb(48, 0, 0);
+  }
+  
+  &:nth-child(6) {
+    background-color: rgb(0, 0, 0);
+  }
```

Comment:
you could use scss loop

Reply:
Can you help me with scss loop please?

Reply:
I done this task without loop (

Reply:
if you need help - ask in fe_chat

Reply:
rgb(51 * index, 0, 0) - but with right syntax